docs/source/conf.py

```diff
@@ -12,18 +12,19 @@
 #
 import os
 import sys
-sys.path.insert(0, os.path.abspath('../..'))
-print('docs path', sys.path[0])
-os.environ['POLYGON_API_KEY'] = 'POLYGON_API_KEY'
+
```

Comment:
ran black on all files; will revert; purpose of this pr is to check direction of changes;